src/mixins/filing-template-mixin.ts

```diff
@@ -586,6 +589,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     return filing
   }
 
+  /** The list of completing parties. */
```

Comment:
Obsolete comment.

Reply:
@shabeeb-aot Please let me resolve the conversations I opened. This allows me to use unresolved conversations to verify code fixes.